enterprise/frontend/src/embedding-sdk/hooks/private/use-init-data.ts

```diff
@@ -39,12 +37,14 @@ export const useInitData = ({ config }: InitDataLoaderParameters) => {
       version: EMBEDDING_SDK_VERSION,
     };
 
-    // eslint-disable-next-line no-console
-    console.log(
-      // eslint-disable-next-line no-literal-metabase-strings -- Not a user facing string
-      `Using Metabase Embedding SDK, version ${EMBEDDING_SDK_VERSION}`,
-    );
-  }, []);
+    if (allowConsoleLog) {
+      // eslint-disable-next-line no-console
+      console.log(
+        // eslint-disable-next-line no-literal-metabase-strings -- Not a user facing string
+        `Using Metabase Embedding SDK, version ${EMBEDDING_SDK_VERSION}`,
+      );
+    }
+  }, [allowConsoleLog]);
```

Comment:
I'm getting a lot of console log spam when running `yarn jest` which is annoying. This allows us (and our customers) to disable console logs.